### PR TITLE
Login

### PR DESCRIPTION
**Pull request by PinkhamSoftware**
What:
- Add to config 
- Switch HttpClient to not use Baseurl which was getting 502

Why:
- Could not login to OutSystems MAT-144

### CODE REVIEW

File: ManageATenancyAPI/Actions/Housing/NHO/TenancyManagementActions.cs [hunk 1]
```diff
@@ -188,6 +188,13 @@ public async Task<TenancyManagement> CreateTenancyManagementInteraction(TenancyM
                 {
                     tmiJObject.Add("hackney_household_Interactionid@odata.bind", " /hackney_households(" + interaction.householdId + ")");
                 }
+
```
Comment (danielwilliams94) [inline ManageATenancyAPI/Actions/Housing/NHO/TenancyManagementActions.cs]: This change is from your other PR, seems like maybe it made it in here by mistake?

File: ManageATenancyAPI/Models/Housing/NHO/TenancyManagement.cs [hunk 2]
```diff
@@ -30,5 +30,6 @@ public class TenancyManagement
         public string householdId { get; set; }
         public int processStage { get; set; }
         public int? reasonForStartingProcess { get; set; }
+        public DateTime? AccountCreatedOn { get; set; }
```
Comment (danielwilliams94) [inline ManageATenancyAPI/Models/Housing/NHO/TenancyManagement.cs]: Same with this, not sure it's supposed to be in this PR

File: ManageATenancyAPI/Actions/Housing/NHO/TenancyManagementActions.cs (hunk 1 above)
Comment (PinkhamSoftware) [reply ManageATenancyAPI/Actions/Housing/NHO/TenancyManagementActions.cs]: Well spotted, resolved.

File: ManageATenancyAPI/Models/Housing/NHO/TenancyManagement.cs (hunk 2 above)
Comment (PinkhamSoftware) [reply ManageATenancyAPI/Models/Housing/NHO/TenancyManagement.cs]: Well spotted, resolved.

**Pull request merged**